erica/application/base_dto.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+
+class BaseDto(BaseModel):
+    pass
```

Comment:
Nice! :+1:

Reply:
I think this was already there but only in another file